MapboxCoreNavigation/RouteController.swift

```diff
@@ -720,6 +722,23 @@ extension RouteController: CLLocationManagerDelegate {
             strongSelf.delegate?.routeController?(strongSelf, didRerouteAlong: route)
         }
     }
+    
+    func checkForUpdates() {
+        #if TARGET_IPHONE_SIMULATOR
+            guard let latestVersion = String(describing: Bundle(for: RouteController.self).object(forInfoDictionaryKey: "CFBundleShortVersionString")) else { return }
+            URLSession.shared.dataTask(with: URL(string: "https://www.mapbox.com/mapbox-navigation-ios/navigation/latest_version")!, completionHandler: { (data, response, error) in
+                if let _ = error { return }
+                guard let httpResponse = response as? HTTPURLResponse, httpResponse.statusCode == 200 else { return }
+                
+                guard let data = data, let currentVersion = String(data: data, encoding: .utf8) else { return }
+                
+                if latestVersion != currentVersion {
```

Comment:
(Note that this string comparison won’t correctly handle any prereleases we do.)

Reply:
Yeah I think thats fine.